extension/flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisExecutionOptions.java

```diff
@@ -28,7 +28,7 @@
 public class DorisExecutionOptions implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    public static final Integer DEFAULT_BATCH_SIZE = 1000;
+    public static final Integer DEFAULT_BATCH_SIZE = 10000;
     public static final Integer DEFAULT_MAX_RETRY_TIMES = 3;
```

Comment:
I think the `DEFAULT_MAX_RETRY_TIMES` can be set to 1 to avoid too many unnecessary retries.
If `DEFAULT_MAX_RETRY_TIMES` is 1, it will only retry once.

Reply:
ok